routers/cart-router.js

```diff
@@ -0,0 +1,93 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const {Cart} = require('../models/cart-model')
+const CartRouter   = express.Router()
+CartRouter.use(bodyParser.json({limit: '5000kb'}))
+const cors = require('cors')
+CartRouter.use(cors())
+CartRouter.route('/')
+.get(async (req , res) =>{
+  try{
+     const myCart = await Cart.find({})
+     res.json({myCart :myCart})
+  }catch(error){
+    res.json({status : false  , message : "couldn't collect the data" , errorMessaage : error.message})
+  }
+ 
+})
+.post(async (req , res ) =>{
+  try{
+    const data = req.body
+    const newProduct =new  Cart(data)
+    const SavedData = await newProduct.save()
+    res.json({SavedData : SavedData})
+  }
+  catch(error){
+    res.json({status : false , message : "failed to get the cart" , errorMessaage : error.message})
```

Comment:
Since you are updating the cart, you can use a different message "Failed to update the cart' instead of "failed to get the cart"